modules/app/components/ClientRenderOnly.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const ClientRenderOnly: React.FC<Props> = ({ children, ...delegated }) => {
```

Comment:
nice idea. what is `delegated` in this case?

Reply:
I guess any props you'd want to pass to the div. We don't need it here but don't think it hurts to include it.